### ISSUE TITLE
Weighting of positive and negative examples

### ISSUE THREAD

**Issue by heindorf** [dice-group/Ontolearn#227]
It would be nice if weights for examples could be specified. The weights can be used, for example, to compute weighted accuracy and F1 measure.

See parameter `sample_weight` in
- https://scikit-learn.org/stable/modules/generated/sklearn.metrics.accuracy_score.html
- https://scikit-learn.org/stable/modules/generated/sklearn.metrics.f1_score.html

**Comment by Demirrr** [comment]
Although it is a very cool idea, I reckon the idea needs to be a deep investigation. Please let me know @heindorf if we start doing a research in this direction :)
